src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -4027,10 +4028,12 @@ private bool IsPossibleParameter(bool allowThisKeyword)
                 case SyntaxKind.OpenBracketToken: // attribute
                 case SyntaxKind.RefKeyword:
                 case SyntaxKind.OutKeyword:
-                case SyntaxKind.OpenParenToken:   // tuple
                 case SyntaxKind.ParamsKeyword:
                 case SyntaxKind.ArgListKeyword:
                     return true;
+                case SyntaxKind.OpenParenToken:   // tuple
+                    return IsFeatureEnabled(MessageID.IDS_FeatureTuples) || IsPredefinedType(this.CurrentToken.Kind);
```

Comment:
I thought about that too. I'll make the change here and the other similar location.